### ISSUE TITLE
pandas stubs - cast Series as float (reportGeneralTypeIssues error)

### ISSUE THREAD

**Issue by tekumara** [microsoft/pylance-release#148]
## Environment data

-   Language Server version: v2020.7.2
-   OS and version: macOS
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.6.10

## Expected behaviour

No type error. At runtime this works, so I would have thought it possible to cast a Series as a float?

## Actual behaviour

```
Argument of type "Series[Unknown]" cannot be assigned to parameter "x" of type "SupportsFloat | _SupportsIndex | str | bytes | bytearray" in function "__init__"
  "__float__" is not present
  "__index__" is not present
  "Series[Unknown]" is incompatible with "str"
  "Series[Unknown]" is incompatible with "bytes"
  "Series[Unknown]" is incompatible with "bytearray"
```

## Code Snippet / Additional information

```python
import pandas as pd

s = pd.Series(1.0)
f = float(s)
```

**Comment by gramster** [comment]
Should be fixed in next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202081-13-august-2020
